docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md

```diff
@@ -0,0 +1,151 @@
+---
+id: cato-networks-source
+title: Cato Networks Source
+sidebar_label: Cato Networks
+description: Learn how to configure the Cato Networks Source Cloud-to-Cloud connector for Sumo Logic.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/cato-logo.png')} alt="img/send-data/cato-logo.png" width="50"/>
+
+Cato Networks is a cloud-native, global SD-WAN provider that delivers a secure, optimized, and agile global network for businesses of all sizes. Cato's cloud-based platform converges multiple network and security functions into a unified solution that includes SD-WAN, network security, cloud security, and secure access service edge (SASE) capabilities.
+
+This source ingests Security and Audit events from Cato API.
+
+## Data Sources
+
+Cato Networks source ingests security events from [EventsFeed API](https://support.catonetworks.com/hc/en-us/articles/360019839477-Cato-API-EventsFeed-Large-Scale-Event-Monitoring) and audit events from [AuditFeed API](https://support.catonetworks.com/hc/en-us/articles/360017900857-Cato-API-AuditFeed).
+
+## Prerequisite
+
+1. Only account administrators with the Editor privilege can generate keys in the Cato Management Application (CMA).
+1. For ingesting security events you need to enable the events feeds in your account. Follow the below-mentioned steps to enable events feed:
+    - In the navigation panel, select **System > API Access Management**.
+    - Select **Event Feed Enabled**. <br/>After this, your account starts sending events to the Cato API server. <br/><img src={useBaseUrl('img/send-data/cato-c2c/cato-networks-enable-events-feed.png')} alt="cato-networks-enable-events-feed.png" width="700" />
```

Comment:
```suggestion
    - Select **Event Feed Enabled**. After this, your account starts sending events to the Cato API server. <br/><img src={useBaseUrl('img/send-data/cato-c2c/cato-networks-enable-events-feed.png')} alt="cato-networks-enable-events-feed.png" width="700" />
```

Reply:
All C2Cs images are saved under the /img/send-data/. No need of separate folders.